models/hh_cond_beta_gap_traub.h

```diff
@@ -89,19 +89,18 @@ to match those used with NEST 1.9.10 when preparing data for [1]_. Code for all
 simulators covered is available from ModelDB [3]_.
 
 Note:
-In this model, a spike is emitted if
-
- :math:`V_m >= V_T + 30 mV` and `V_m` has fallen during the current time step
+In this model, a spike is emitted if :math:`V_m \geq V_T + 30` mV and :math:`V_m` has fallen during the current time
+step.
 
 To avoid that this leads to multiple spikes during the falling flank of a
 spike, it is essential to chose a sufficiently long refractory period.
-Traub and Miles used \f$ t_ref = 3 ms \f$ [2, p 118], while we used
-\f$ t_ref = 2 ms \f$ in [2]_.
+Traub and Miles used :math:`t_{ref} = 3` ms ([2]_, p 118), while we used
+:math:`t_{ref} = 2` ms in [2]_.
 
 Post-synaptic currents
 Incoming spike events induce a post-synaptic change of conductance modelled by a
-beta function as outlined in [4,5]. The beta function is normalised such that an
-event of weight 1.0 results in a peak current of 1 nS at \f$ t = tau_rise_xx \f$
+beta function as outlined in [4]_ [5]_. The beta function is normalised such that an
```

Comment:
Could we change the few instances of "normalised" to "normalized"?

Reply:
Done!